apps/web/src/config/index.ts

```diff
@@ -23,7 +23,7 @@ export const ENV = process.env.REACT_APP_ENVIRONMENT;
 
 export const APP_ID = process.env.REACT_APP_NOVU_APP_ID;
 
-export const WIDGET_EMEBED_PATH = process.env.REACT_APP_WIDGET_EMBED_PATH || 'http://localhost:4701/embed.umd.min.js';
+export const WIDGET_EMBED_PATH = process.env.REACT_APP_WIDGET_EMBED_PATH || 'http://localhost:4701/embed.umd.min.js';
```

Comment:
Rollback this one please. This can affect current users settings and break their systems.